src/test/java/com/thirdcc/webapp/web/rest/EventAttendeeResourceIT.java

```diff
@@ -532,29 +532,19 @@ public void getEventAttendeeByEventIdAndUserId_withNonExistingUser_ShouldReturnO
 
         //Save events and attendee
         restEventAttendeeMockMvc.perform(get("/api/event-attendees/event/{eventId}/user/{userId}", savedEvent.getId(), Long.MAX_VALUE ))
-            .andExpect(status().isOk())
-            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
-            .andExpect(jsonPath("$.id").value(nullValue()))
-            .andExpect(jsonPath("$.userId").value(nullValue()))
-            .andExpect(jsonPath("$.eventId").value(nullValue()))
-            .andExpect(jsonPath("$.provideTransport").value(nullValue()));
```

Comment:
Why not keep this?

Reply:
Resolved in c9e67c7c1113b73f838af40f9d8987ca995becae